openmdao/utils/om.py

```diff
@@ -453,6 +507,8 @@ def _cite(prob):
     'cite': (_cite_setup_parser, _cite_cmd, 'Print citations referenced by the problem.'),
     'compute_entry_points': (_compute_entry_points_setup_parser, _compute_entry_points_exec,
                              'Compute entry point declarations to add to the setup.py file.'),
+    'debug': (_show_dependency_versions_parser, _show_dependency_versions,
+              'Show dependency versions'),
```

Comment:
`openmdao debug` by itself doesn't do anything...

I thought idea was to add the `--show_dependencies` argument alongside `--version` in the `openmdao_cmd()` function...